src/components/tuCollapse/tuCollapseItem.vue

```diff
@@ -1,11 +1,7 @@
 <template>
-	<div
-		:class="{ 'open-item': maxHeight != '0px', disabledx: disabled }"
-		class="tu-collapse-item"
-		@mouseover="mouseover"
-		@mouseout="mouseout"
-	>
-		<header class="tu-collapse-item--header" @click="toggleContent">
+	<details :open="maxHeight != '0px' ? true : false">
```

Comment:
The conditional operator already returns a boolean value. You do not need explicitly return a boolean
Please simplify this to `maxHeight != '0px'`